_posts/2021-09-28-p2pool-released.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Monero P2Pool is now live"
+summary: P2Pool decentralized pool for Monero mining is live on the mainnet
+tags: [community]
+author: QuickBASIC
+---
+
+We are happy to announce that Monero P2Pool is live on the Monero network and binaries have been [released](https://github.com/SChernykh/p2pool/releases).
+
+### What is Monero P2Pool?
+
+Monero P2Pool is a peer-to-peer Monero mining pool developed from scratch by SChernykh (also known as sech1). P2Pool combines the advantages of pool and solo mining; you still fully control your Monero node and what it mines, but you get frequent payouts like on a regular pool.
+P2Pool as a concept was first developed for the Bitcoin blockchain but was never fully realized due to certain limitations, mainly because it had a problem with orphaned blocks which is solved in Monero P2Pool with uncle blocks.
+
+P2Pool has no central server that can be shutdown/blocked because it uses a separate blockchain to merge mine with Monero. There's no pool admin that can control what your hashrate is used for or decide who can mine on the pool and who can't --- It's permissionless!
```

Comment:
this is a stylistic change, but I recommend a semicolon over `---` for reasons clearer on line 20

Reply:
Ack